packages/app/src/ui/pages/OnboardingBackup/OnboardingBackup.tsx

```diff
@@ -0,0 +1,90 @@
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+
+import { Icon } from "@src/ui/components/Icon";
+import { PasswordInput } from "@src/ui/components/PasswordInput";
+import { UploadInput } from "@src/ui/components/UploadInput/UploadInput";
+
+import { useOnboardingBackup } from "./useOnboardingBackup";
+
+const OnboardingBackup = (): JSX.Element => {
+  const { isShowPassword, isLoading, errors, register, onDrop, onGoBack, onShowPassword, onSubmit } =
+    useOnboardingBackup();
+
+  return (
+    <Box
+      data-testid="onboarding-backup-page"
+      sx={{ display: "flex", flexDirection: "column", flexWrap: "nowrap", height: "100%" }}
+    >
+      <Box sx={{ display: "flex", flexDirection: "column", p: 2, flexGrow: 1 }}>
+        <Box sx={{ display: "flex", alignItems: "center", justifyContent: "space-between" }}>
+          <Typography variant="h4">Upload backup</Typography>
+
+          <Icon data-testid="close-icon" fontAwesome="fas fa-times" size={1.25} onClick={onGoBack} />
+        </Box>
+
+        <Box
+          component="form"
+          sx={{
+            mt: 2,
+            display: "flex",
+            flexDirection: "column",
+            flexWrap: "nowrap",
+            flexGrow: 1,
+          }}
+          onSubmit={onSubmit}
+        >
+          <Box sx={{ display: "flex", flexDirection: "column", alignItems: "center", flexGrow: 1, width: "100%" }}>
+            <Typography fontWeight="bold" variant="body1">
+              To upload your backup, please provide backup file and backup password.
+            </Typography>
+
+            <Typography sx={{ my: 1, alignSelf: "flex-start" }} variant="body1">
+              Note: backup will use backup password as your login password and mnemonic phrase from backup file
```

Comment:
I think we should do this in a yellow warning box, or green info box. But I mean to make it more notifiable to the user. 
![Screenshot_20230728_160114](https://github.com/CryptKeeperZK/crypt-keeper-extension/assets/36774944/2df7bbaa-8508-4415-b451-1dcd450962c9)